src/OpenLoco/src/S5/Limits.h

```diff
@@ -15,6 +15,7 @@ namespace OpenLoco::S5::Limits
     constexpr size_t kMaxUserStrings = 2048;
     constexpr size_t kMaxVehicles = 1000;
     constexpr size_t kMaxRoutingsPerVehicle = 64;
+    constexpr size_t kMaxOrdersPerVehicle = 63;
```

Comment:
suspiciously similar to kMaxRoutingsPerVehicle. Are we sure this is not the same thing?

Reply:
I considered using that constant as well. It appears to be used here:
```
uint16_t routings[Limits::kMaxVehicles][Limits::kMaxRoutingsPerVehicle]; // 0x442A44 (0x0096885C)
```
I suspect the constant 63 is used for order tables so the 64th order is always a nullbyte (OrderEnd type)

Reply:
I suggest we define this then as kMaxRoutingsPerVehicle - 1 and add a note that there is a null (OrderEnd)